### ISSUE TITLE
Reverse bit numbering in IBCM documentation

### ISSUE THREAD

**Issue by cceckman** [aaronbloomfield/pdr#21]
**TL;DR IBCM bits are numbered 0 to 15 from left to right; should be 15 down to 0.**

The IBCM documentation uses a bit numbering convention where bits are numbered left-to-right. In all of the embedded documentation I've seen, however, a right-to-left convention is used. 

According to the IBCM doucmentation,
0x1080 = 0b 0001 0000 1000 0000
would have bits 3 and 8 set and all others clear; bit 0 is the most-significant bit, and bit 15 is the least-significant. In the convention that ARM and Intel use in their documentation, bit 0 is the least significant, and bit 15 the most significant; the same convention is used in our school's CpE sequence.

This choice is not simply arbitrary; it has implications for the numeric interpretation. In a right-to-left numbering, bit 0 represents 2^0, bit 1 represents 2^1, etc. There's no such correspondence with a left-to-right numbering.

Proposal: revise the IBCM slide set, chapter, etc. so that students are used to seeing bits run from (N-1) down to 0. This is in keeping with conventions of professional documentation and the numeric meaning.
